src/include/duckdb/storage/statistics/node_statistics.hpp

```diff
@@ -23,6 +24,28 @@ class NodeStatistics {
 	    : has_estimated_cardinality(true), estimated_cardinality(estimated_cardinality), has_max_cardinality(true),
 	      max_cardinality(max_cardinality) {
 	}
+	void Serialize(Serializer &serializer) const { 
+		serializer.Write(has_estimated_cardinality);
+		if (has_estimated_cardinality) {
+			serializer.Write(estimated_cardinality);
+			serializer.Write(has_max_cardinality);
+			if (has_max_cardinality) {
+				serializer.Write(max_cardinality);
+			}
+		} else {
+			assert(!has_max_cardinality);
```

Comment:
`D_ASSERT()` instead of `assert`

Reply:
ah, I saw it did not even survive the CI debug build (but it did on mine, does not compute). So fixed it. And I also ran make format-fix now